replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+#View(dat_egood)
+
+## Redo Chi-Square with subsetted data:
+
+# Test whether music condition is independent of pen choice:
+# Create 2x2 contingency table of frequencies - pen choice by music condition
+tbl2 <- table(dat_egood$music, dat_egood$chose_advertised_pen) 
+tbl2    
+
+# Chi-Square, no correction, no simulation:
+cs2 <- chisq.test(tbl2, correct = FALSE)
+tidy(cs2)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .35, N = 72, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with minimum of .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 72, df = 1, sig.level = 0.05, power = .8)
+
+# Confidence Intervals (not replicated from paper):
+#upper (95%)
+#CI95_up <- qchisq(.95, df = 1)
+#lower (5%)
+#CI95_low <- qchisq(.05, df = 1)
+
+## Replicate Table: Frequency table & chi-square statistic of subsetted data
+# (replicate "Experiment 3" portion of the table from article)
+
+# Generate count and proportion (percent) tallies for replicating the table:
+# Amongst those in Liked Music condition:
+dat_egoodLike <- dat_egood %>%
+  filter(music == 1)
+dat_egoodLike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodLike) * 100)
+
+# Amongst those in Disliked Music condition:
+dat_egoodDislike <- dat_egood %>%
+  filter(music == 2)
+dat_egoodDislike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodDislike) * 100)
+
+# Generate table:
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
+Disliked_Music <- c("8(23%)","27(77%)", "")
+Condition <- c("AdvertisedPen", "NonAdvertisedPen", "Chi-Square")
+Freq_Table <- cbind(Condition, Liked_Music, Disliked_Music)
+stargazer(Freq_Table, type = "text", title = "Study 3: Frequencies of Pen Choice for Each Conditon & Chi-Square")
+
+```
+
+(4.1) Extension A: Graphing the frequency of response & Investigating moderation by gender:
+
+```{r echo = FALSE}
+
+ggplot(dat_egood, 
+       aes(x = music, fill = chose_advertised_pen)) +
+  geom_bar(stat = "bin",
+           position = position_dodge()) + 
+  scale_fill_brewer(name = "Pen Choice", 
+                    labels = c("Not Advertised", "Advertised"), palette = "Set2") +
+  ggtitle("Frequency of Pen Choice By Condition") +
+  ylab("Frequency") +
+  xlab("Music Conditon") +
+  scale_x_discrete(breaks=c("1", "2"), 
+                   labels=c("Liked", "Disliked")) +
+  theme_bw() + 
+  theme(axis.title.x = element_text(face = 'bold', size = 15),
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+```
```

Comment:
Might be a good idea to save these graphs both as images as well as with object names.